.github/workflows/mpc-test-ci.yaml

```diff
@@ -0,0 +1,66 @@
+name: Validate PR - Testing Phase - golang CI
```

Comment:
this name should probably indicate this would run SeaLights.

Reply:
Fixed in new commit